go.mod

```diff
@@ -21,27 +21,40 @@ require (
 )
 
 require (
+	github.com/NYTimes/gziphandler v1.1.1 // indirect
```

Comment:
Do you know how much larger this makes the binary?

It seems to be a lot of dependencies we brought in..

Reply:
I'm not sure, I'll have to look into it.